source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,158 @@
+.. _scala-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:atlas:`Atlas Search </atlas-search>` enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. Atlas Search indexes specify the behavior of
+the search and which fields to index.
+
+You can call the following methods on a collection to manage your Atlas Search
+indexes:
+
+- ``createSearchIndex()``
+- ``createSearchIndexes()``
+- ``listSearchIndexes()``
+- ``updateSearchIndex()``
+- ``dropSearchIndex()``
+
+.. note::
+
+   The Atlas Search index management methods run asynchronously and might return before
+   confirming that they ran successfully. To determine the current status of the indexes,
+   call the ``listSearchIndexes()`` method.
+
+The following sections provide code examples that demonstrate how to use
+each of the preceding methods.
+
+.. _scala-atlas-search-index-create:
+
+Create a Search Index
+---------------------
+
+You can use the ``createSearchIndex()`` and the ``createSearchIndexes()`` methods to
+create one or more Atlas Search indexes.
+
+You can also use these methods to create Atlas Vector Search Indexes.
+Atlas Vector Search enables you to perform semantic searches on vector
+embeddings stored in MongoDB Atlas. To learn more about this feature,
+see the :atlas:`Atlas Vector Search Overview
+</atlas-vector-search/vector-search-overview/>`.
+
+The following code example shows how to create an Atlas Search index:
```

Comment:
It's not clear why you don't need to specify an index name here, but you do when creating multiple indexes.  Can we clarify that in the text?